java/client/src/main/java/glide/api/models/commands/LmPopOptions.java

```diff
@@ -0,0 +1,23 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.ListBaseCommands;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Enumeration representing element pop direction and count for the {@link ListBaseCommands#lmpop}
+ * command.
+ */
+@RequiredArgsConstructor
+@Getter
+public enum LmPopOptions {
+    /** Represents the option that elements shoot be pop from the left side of a list */
+    LEFT("LEFT"),
+    /** Represents the option that elements shoot be pop from the right side of a list */
+    RIGHT("RIGHT"),
+    /** Represents the COUNT indicator that comes before the count argument */
+    COUNT("COUNT");
+
+    private final String redisApi;
+}
```

Comment:
Simplify that and use `toString()` and update docs
```suggestion
public enum LmPopOptions {
    /** Represents the instruction to pop elements from the left side of a list. */
    LEFT,
    /** Represents the instruction to pop elements from the right side of a list. */
    RIGHT,
}
```
Don't store `COUNT` there, because a user may use that.

Reply:
I think you can do
```
interface LmPopOptions {
  string COUNT
  enum Direction {
    L, R
  }
}